doc/upgrade-guide/README.md

```diff
@@ -9,6 +9,12 @@ If you have any questions or comments, you can [post them on the mailing list][m
 
 The following is a list of changes made in version 4.0 of the driver that are relevant when upgrading from version 3.x.
 
+### localDataCenter is now a required Client option
```

Comment:
👍 on adding this info first in the upgrade guide.

I'll adapt it on the DSE driver side.